docs/ispc.rst

```diff
@@ -674,6 +674,8 @@ Updating ISPC Programs For Changes In ISPC 1.26.0
 Macro definitions for LLVM version that ISPC is based on were added. Please,
 refer to `The Preprocessor`_ for more details.
 
+``--nocpp`` command line flag is deprecated and will be removed in future.
+
```

Comment:
There is a mention of `--nocpp` flag in Preprocessor section - I think it can be removed.

Reply:
Done

Reply:
I don't see ispc.rst in the modified files anymore.

Reply:
Right, it has to be now.